caddyconfig/httpcaddyfile/httptype.go

```diff
@@ -43,6 +43,16 @@ func (st ServerType) Setup(originalServerBlocks []caddyfile.ServerBlock,
 	var warnings []caddyconfig.Warning
 	gc := counter{new(int)}
 
+	state := make(map[string]interface{})
```

Comment:
Maybe even bump this up a line so it's together with warnings and gc, which are also state we share in a very similar way.

Reply:
Done